run.py

```diff
@@ -140,6 +144,25 @@ def generate_app(
         allow_headers=["*"],
     )
 
+    @app.middleware("http")
+    async def block_origin_middleware(request: Request, call_next):
```

Comment:
経緯を知らない人でも目的が一瞬でわかるよう、なにかコメント書くのはどうでしょう。
例えばこんな感じとか･･･！
```suggestion
    # 許可されていないoriginを遮断する
    @app.middleware("http")
    async def block_origin_middleware(request: Request, call_next):
```